TypeCobol/Compiler/SqlScanner/SqlChar.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TypeCobol.Compiler.SqlScanner
+{
+    /// <summary>
+    /// Check the possible role of Sql characters in source text
+    /// </summary>
+    public static class SqlChar
```

Comment:
For this single method, I think it could be moved into `SqlScanner`.

Reply:
done